src/test/java/teammates/test/cases/common/ActivityLogEntryTest.java

```diff
@@ -47,6 +47,11 @@ public void testActivityLogEntryClass() {
     public void testGetActionName(){
         assertEquals("instructorCourse", ActivityLogEntry.getActionName("/page/instructorCourse"));
         assertEquals("instructorCourse", ActivityLogEntry.getActionName("/page/instructorCourse?user=x"));
-        assertEquals("error in getActionName for requestUrl : instructorCourse", ActivityLogEntry.getActionName("instructorCourse"));
+        try {
+            ActivityLogEntry.getActionName("instructorCourse");
+            signalFailureToDetectException("getActionName should throw an exception if an action cannot be retrieved");
+        } catch (Exception e) {
```

Comment:
I believe this is specifically `ArrayIndexOutOfBoundsException`. Try not to catch a generic one.